cfg/poms/official/MCP2021C_test/MCP2021C_test-gen_g4_detsim_reco1.cfg

```diff
@@ -0,0 +1,201 @@
+[global]
+group      = sbnd
+experiment = sbnd
+sbndpro    = sbndpro
+wrapper    = file:///${FIFE_UTILS_DIR}/libexec/fife_wrap
+version    = v09_37_01
```

Comment:
here and elsewhere, what is the `version` used for? with a patch release the tag will change (most likely to v09_37_01_01)

Reply:
@Li-Jiaoyang97 made the changes to keep templates of the .cfg file so that version can be manually filled in.